Sources/AblyChat/Room.swift

```diff
@@ -105,26 +150,38 @@ internal actor DefaultRoom<LifecycleManagerFactory: RoomLifecycleManagerFactory>
             logger: logger
         )
 
-        _reactions = options.reactions != nil ? await DefaultRoomReactions(
-            featureChannel: featureChannels[.reactions]!,
-            clientID: clientId,
-            roomID: roomID,
-            logger: logger
-        ) : nil
+        _reactions = if let featureChannel = featureChannels[.reactions] {
```

Comment:
Is it swift 6 construct?

Reply:
if-as-expression was added recently, yeah, not sure exactly which Swift version